test/box/several-listening-sockets.result

```diff
@@ -0,0 +1,173 @@
+-- test-run result file version 2
+test_run = require('test_run').new()
+ | ---
+ | ...
+net_box = require('net.box')
+ | ---
+ | ...
+fio = require('fio')
+ | ---
+ | ...
+
+default_listen_addr = box.cfg.listen
+ | ---
+ | ...
+test_run:cmd("setopt delimiter ';'")
+ | ---
+ | - true
+ | ...
+-- Create table which contain several listening uri,
+-- according to template @a addr. It's simple adds
+-- capital letters in alphabetical order to the
+-- template.
+function create_uri_table(addr, count)
+    local uris_table = {}
+    local path_table = {}
+    local ascii_A = 97
+    for i = 1, count do
+        local ascii_code = ascii_A + i - 1
+        local letter = string.upper(string.char(ascii_code))
+        path_table[i] = addr .. letter
+        uris_table[i] = "unix/:" .. addr .. letter
+    end
+    return uris_table, path_table
+end;
+ | ---
+ | ...
+function check_connection(port)
+    local conn = net_box.connect(port)
+    assert(conn:ping())
+    conn:close()
+end;
+ | ---
+ | ...
+test_run:cmd("setopt delimiter ''");
+ | ---
+ | - true
+ | ...
+
+-- Check connection if listening uri passed as a single port number.
+port_number = 0
+ | ---
+ | ...
+box.cfg{listen = port_number}
+ | ---
+ | ...
+check_connection(box.info.listen)
+ | ---
+ | ...
+
+-- Check connection if listening uri passed as a single string.
+unix_socket_path = "./tarantool" .. "A"
```

Comment:
Too much formatting and concatenations, it could be much simpler.
```Diff
 -- Check connection if listening uri passed as a single string.
-unix_socket_path = "./tarantool" .. "A"
-box.cfg{listen = "unix/:" .. unix_socket_path}
-assert(box.cfg.listen == "unix/:" .. unix_socket_path)
+unix_socket_path = "unix/:./tarantoolA"
+box.cfg{listen = unix_socket_path}
+assert(box.cfg.listen == unix_socket_path)
 assert(box.info.listen:match("unix/:"))
-check_connection("unix/:" .. unix_socket_path)
+check_connection(unix_socket_path)
```